instrumentation/jvm-metrics/build.gradle.kts

```diff
@@ -7,38 +7,7 @@ dependencies {
   compileOnly("io.opentelemetry.javaagent:opentelemetry-javaagent-tooling")
 }
 
-tasks {
-  val testMicrometerJvmMetrics by registering(Test::class) {
-    filter {
-      includeTestsMatching("*MicrometerJvmMetricsTest")
-    }
-    include("**/*MicrometerJvmMetricsTest.*")
-    jvmArgs("-Dsplunk.metrics.implementation=micrometer")
-  }
-
-  val testOtelJvmMetrics by registering(Test::class) {
-    filter {
-      includeTestsMatching("*OtelJvmMetricsTest")
-    }
-    include("**/*OtelJvmMetricsTest.*")
-    jvmArgs("-Dsplunk.metrics.implementation=opentelemetry")
-  }
-
-  test {
-    filter {
-      excludeTestsMatching("*MicrometerJvmMetricsTest")
-      excludeTestsMatching("*OtelJvmMetricsTest")
-      isFailOnNoMatchingTests = false
-    }
-  }
-
-  test {
-    dependsOn(testMicrometerJvmMetrics)
-    dependsOn(testOtelJvmMetrics)
-  }
-
-  withType<Test>().configureEach {
-    jvmArgs("-Dsplunk.metrics.enabled=true")
-    jvmArgs("-Dsplunk.metrics.experimental.enabled=true")
-  }
+tasks.withType<Test>().configureEach {
+  jvmArgs("-Dsplunk.metrics.enabled=true")
+  jvmArgs("-Dsplunk.metrics.experimental.enabled=true")
```

Comment:
`splunk.metrics.experimental.enabled` isn't mentioned in `metrics.md` yet? I didn't see it. Do we need to include that?

Reply:
I think the only purpose of this flag is to enable testing these metrics. I think we don't need to document this unless there is someone that wishes to use these metrics without having memory profiling enabled. As far as I understand these metrics are there only to be used in memory profiling charts. I guess if we wanted we could delete this flag and use the memory profiling flag here instead.